frontend/app/src/components/ModalContent.svelte

```diff
@@ -28,14 +28,22 @@
     export let actualWidth: number = 0;
     export let closeIcon: boolean = false;
     export let square: boolean = false;
+    export let width: number | undefined = undefined;
 
     // if your modal *definitely* overflows on mobile you might need to set height explicitly
     export let overflows: boolean = false;
 
     let divElement: HTMLElement;
+    let style = "";
 
     $: useAlignTo = alignTo !== undefined && !$mobileWidth;
-    $: style = useAlignTo ? "visibility: hidden;" : "visibility: visible;";
+    $: {
```

Comment:
this seems quite fishy. I would have thought that it's actually _not_ the modal that is fixed width, it is (probably always) the modal's content. So you're actually saying, the captcha should be 400px wide and the modal should just be as wide as it needs to be to show its contents.